test/integration/app.js

```diff
@@ -34,194 +30,358 @@ describe('redirection', () => {
     chai.request(server)
       .get(constants.SITE_ROOT)
       .end((err, res) => {
-        checkHtmlResponse(err, res);
+        iExpect.checkHtmlResponse(err, res);
+
+        // eslint-disable-next-line no-unused-expressions
         expect(res).to.redirect;
         expect(res.req.path).to.equal(`${constants.SITE_ROOT}/find-help`);
         done();
       });
   });
 });
 
-describe('The stomach ache page', () => {
-  it('should be accessible directly', (done) => {
+describe('An unknown page', () => {
+  it('should return a 404', (done) => {
     chai.request(server)
-      .get(`${constants.SITE_ROOT}/stomach-ache`)
+      .get(`${constants.SITE_ROOT}/not-known`)
       .end((err, res) => {
-        checkHtmlResponse(err, res);
+        expect(err).to.not.be.equal(null);
+        expect(res).to.have.status(404);
+        // eslint-disable-next-line no-unused-expressions
+        expect(res).to.be.html;
 
         const $ = cheerio.load(res.text);
 
-        expect($('.local-header--title').text())
-          .to.equal('Stomach ache');
+        expect($('.local-header--title--question').text().trim())
```

Comment:
Just for info I've raised an issue regarding the class naming https://github.com/ermlikeyeah/cat-static-dev/issues/2